.github/workflows/performance-test.yaml

```
@@ -18,7 +18,8 @@ jobs:
         # currently we have a duplication, add your benchmark names here and in `performance-test-upload`
         name: ['artificial', 'social-science']
     with:
-      action: performance-test -- 1 "${{ matrix.name }}"
+      # 1 parallel process, 4 runs
+      action: performance-test -- 1 4 "${{ matrix.name }}"
```

```suggestion
      # 1 parallel process, 5 runs
      action: performance-test -- 1 5 "${{ matrix.name }}"
```

Let's use a common number :D

---

you said three to five so i thought: okay